src/test/java/it/eng/idsa/businesslogic/util/TestUtilMessageService.java

```diff
@@ -17,57 +17,67 @@
 import de.fraunhofer.iais.eis.Message;
 import it.eng.idsa.multipart.processor.MultipartMessageProcessor;
 
-/**
- * Util class used for testing
- * Create methods for generating classes when required
- */
 public class TestUtilMessageService {
 
-	private static URI REQUESTED_ARTIFACT = URI.create("http://mdm-connector.ids.isst.fraunhofer.de/artifact/1");
 
-	private static URI ISSUER_CONNECTOR = URI.create("http://true.engineering.it/ids/mdm-connector");
+	public static URI REQUESTED_ARTIFACT = URI.create("http://mdm-connector.ids.isst.fraunhofer.de/artifact/1");
 
-	private static String MODEL_VERSION = "4.0.0";
-
-	private static XMLGregorianCalendar ISSUED;
+	public static URI ISSUER_CONNECTOR = URI.create("http://true.engineering.it/ids/mdm-connector");
```

Comment:
Can we use http://w3id.org/engrd/connector/ url?

Reply:
Sure, no problem.